### PR TITLE
whoops

### PR DESCRIPTION
**Pull request by 2Blagden**
sorted key issue out

### CODE REVIEW

File: lib/takeaway.rb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 require 'twilio-ruby'
 
 # put your own credentials here
-account_sid = 'AC71ede767f3f69bf43718d8094b244800'
-auth_token = '88ac7f43909c00b5b8e9a32d5b554c35'
+#taken them out for pull request
```
Comment (david-mears) [inline lib/takeaway.rb]: Deleting them doesn't quite work for security - I can still find them here! Once you have got feedback from a coach on this repo, I'd consider making it a private repo.

File: lib/takeaway.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+require 'twilio-ruby'
+
+# put your own credentials here
```
Comment (david-mears) [inline lib/takeaway.rb]: A good instruction to add to your README.

File: lib/takeaway.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+class Menu
+  def initialize
+    @list_of_dishes = []
+  end
+  def list_of_dishes
+    @list_of_dishes
+  end
```
Comment (david-mears) [inline lib/takeaway.rb]: This is fine; you could also use `attr_reader :list_of_dishes` to achieve the same effect.

Comment (david-mears) [reply lib/takeaway.rb]: Also, there's a way to build the program that doesn't involve `Menu` spilling its secrets (= allowing other objects to directly access its list of dishes): [see the exemplar](https://github.com/makersacademy/takeaway-exemplar/blob/master/lib/menu.rb).

File: spec/takeaway_spec.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+describe Menu do
+  it 'shows list of dishes' do
+  end
```
Comment (david-mears) [inline spec/takeaway_spec.rb]: Did you forget to write this test, because it showed as 'passing'? If you run a
```ruby
it "blah blah" do
end
```
with nothing in it, it will show up as passing. You can tag it as pending by using `xit` instead of `it`.

File: lib/takeaway.rb [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  ordername = gets.chomp
+  p("quantity:")
+  orderquan = gets.chomp.to_i
+  order << ordername
+  order << orderquan
+  process_order(order)
+end
+
+def process_order(order)
+end
```
Comment (david-mears) [inline lib/takeaway.rb]: All these methods - `send_a_text`, `show_menu`, `make_order` - should live on a class. Think about what classes best suit the responsibilities of these methods.

**Comment by david-mears** [comment]
The two most important things for me are [this issue](https://github.com/makersacademy/takeaway-challenge/pull/1460/commits/c859d607da5a2e124234cf1450c9d2843c09ed48#r335339216) and completing the tests (the `make_order` and `show_menu` methods need tests, and the test for the first user story is empty). For any new test or method, make a commit with the new test first, and then a commit where the test has been passed by the code. This would be good evidence for good progress (green) in OOP. For excellent progress, test-drive a method that returns the total price for the order. Ping me when you've done this.
